test/Altinn.Platform.Authentication.Tests/Controllers/AuthenticationControllerTests.cs

```diff
@@ -1337,7 +1337,8 @@ public async Task AuthenticateEndUser_RequestTokenWithValidExternalToken_Returns
             claims.Add(new Claim("pid", pid));
             claims.Add(new Claim("amr", amr));
             claims.Add(new Claim("acr", acr));
-
+            claims.Add(new Claim("scope", "altinn:instances.read"));
+;
```

Comment:
extra semi colon

Reply:
ugh. Updated scope just before checkin in.